pkg/providers/pricing/initial-on-demand-prices.json

```diff
@@ -1721,6 +1729,10 @@
         "ecs.i3g.2xlarge": 2.8,
         "ecs.i3g.4xlarge": 5.6,
         "ecs.i3g.8xlarge": 11.2,
+        "ecs.i4r.16xlarge": 0,
+        "ecs.i4r.32xlarge": 0,
```

Comment:
quite strange, we should check this, why it's zero

Reply:
ok, we'll troubleshoot this later, here, let's backtrack.